modules/gathering-data-specific-features.adoc

```diff
@@ -45,6 +45,9 @@ endif::openshift-rosa,openshift-dedicated[]
 |`registry.redhat.io/openshift-logging/cluster-logging-rhel9-operator:v<installed_version_logging>`
 |Data collection for {logging}.
 
+|`image=quay.io/netobserv/must-gather`
```

Comment:
I dont know if the `image=` is required. I see that the table description mentions to pass this image with the `--image` option

Reply:
Oh great catch!